how-to.md

```diff
@@ -134,10 +134,10 @@ You can verify the licenses in your tenant [in the Azure Portal](https://portal.
 
 ## Revert your tenant to a point in time
 
-* :warning: **Warning**	:warning: this is a destructive operation that can result in deleting data (such as users) from your tenant(s). Please carefully review the pending changes on the Sync page before approving
```

Comment:
Is there a reason you decided to strike " Please carefully review the pending changes on the Sync page before approving", @brad-Simeon ?

Reply:
This sentence was moved from the top warning to the last step of the instructions, however I am fine to keep it in the warning to make sure the warning is strong.